.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,131 @@
+---
+name: Build and Publish
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
This was a fairly "stock" concurrency string that I was copying. I think you're right in this case, though there is no harm in keeping the pull request number since if it's not a PR that field will basically be ignored.